obspy/io/iaspei/tests/test_core.py

```diff
@@ -28,17 +28,14 @@ def _assert_catalog(got):
     data, num_subs = re.subn(expected_id_prefix, got_id_prefix, data)
     # 49 resource id replacements should be done in the QuakeML file we compare
     # against
-    if num_subs != 49:
-        raise AssertionError()
+    assert num_subs == 49
```

Comment:
So `assert` is OK for test code I guess?

Reply:
Yea I would say so. `pytest` bases everything on top of assert statements and I really know nobody who runs python with `-O`.

Reply:
Alright.. just thought I'd remembered something about better avoiding it.. (from you ;-P but probably just imagining things ;-))